dashboard/app/helpers/levels_helper.rb

```diff
@@ -237,7 +237,7 @@ def question_options
     level_options[:lastAttempt] = @last_attempt
     level_options.merge! @level.properties.camelize_keys
 
-    if current_user.nil? || current_user.teachers.empty?
+    if !@script.try(:professional_learning_course) && (current_user.nil? || current_user.teachers.empty?)
       # only students with teachers should be able to submit
```

Comment:
This comment isn't really applicable, or it misleading now that this is getting exercised for teachers in PD.